src/main/res/values/market.xml

```diff
@@ -114,6 +114,13 @@
     * Use "smart" quotation marks (eg: ’).
     * The full changelog is only in English.
     -->
+    <string name="market_changelog_300">• Adds support for Cifial, Fribourg TPF, Waltti and blue HSL cards\n• Bug fixes for Amiibo, Bonobus, SLaccess, Troika and Viva\n• New data for Adelaide Metrocard, Compass, Gautrain, Rav-Kav, RicaricaMi, ORCA, OùRA, Touch ’n Go and Viva\n• Adds history search, MIFARE Plus support\n• Bug fixes on Android 4.3 and older\n\nMore details and known issues (in English): https://www.metrodroid.org/changelog</string>
```

Comment:
Bonobus is not bugs but additional parsing

Reply:
Fixed in 3aa740d5c